tests/FSharp.Compiler.ComponentTests/ErrorMessages/ConfusingTypeName.fs

```diff
@@ -10,7 +10,7 @@ open FSharp.Compiler.SourceCodeServices
 
 module ``Confusing Type Name`` =
 
-    [<Fact>]
+    [<Fact (Skip = "Test platform C# build fails with latest fsharp.core")>]
```

Comment:
why skip?

Reply:
I mentioned this at standup today, the test harness has issues with building C# libraries.  Vlad has made some changes in master that should improve things.  Anyway, I will work with vlad to re-enable these and the other test cases skipped because of C# test compilation.  But it's not a product issue.

Reply:
Gotcha, makes sense.